pkgs/development/python-modules/kinparse/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonPackage {
+  pname = "kinparse";
+  version = "unstable-2019-12-18";
+
+  src = fetchFromGitHub {
+    owner = "xesscorp";
+    repo = "kinparse";
+    rev = "eeb3f346d57a67a471bdf111f39bef8932644481";
+    sha256 = "1nrjnybwzy93c79yylcwmb4lvkx7hixavnjwffslz0zwn32l0kx3";
+  };
+
+  doCheck = true;
+  pythonImportsCheck = [ "kinparse" ];
+
+  checkInputs = with python3Packages; [
+    pytest
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    future
+    pyparsing
+  ];
+
+  meta = with stdenv.lib; {
```

personal preference to avoid stdenv when possible
```suggestion
  meta = with lib; {
```